docs/user/neuralnet.rst

```diff
@@ -317,43 +330,50 @@ predict(X) and predict_proba(X)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 These methods perform an inference step on the input data and return
-``numpy array``\s. By default, ``predict_proba`` will return whatever
-it is that the ``module``\'s ``forward`` method returns, cast to a
-``numpy array``. If ``forward`` returns multiple outputs as a tuple,
+``numpy array``\s. By default,
+:func:`~skorch.net.NeuralNet.predict_proba` will return whatever it is
+that the ``module``\'s :func:`~torch.nn.Module.forward` method
```

Comment:
Here and in the next paragraph, please check if I picked correct classes' forward/forward_iter methods.

Reply:
I didn't catch any error here.